### PR TITLE
use term package instead of gopass

### PR DESCRIPTION
**Pull request by umezawatakeshi**
part of https://github.com/cybozu-go/cke/issues/484

Signed-off-by: UMEZAWA Takeshi <takeshi-umezawa@cybozu.co.jp>

### CODE REVIEW

File: pkg/ckecli/cmd/vault_init.go [hunk 1]
```diff
@@ -59,6 +60,30 @@ path "cke/*"
...
+	screen := struct {
+		io.Reader
+		io.Writer
+	}{os.Stdin, os.Stdout}
+
+	oldState, err := term.MakeRaw(int(os.Stdin.Fd()))
+	if err != nil {
+		return "", err
+	}
+	defer term.Restore(0, oldState)
```
Comment (morimoto-cybozu) [inline pkg/ckecli/cmd/vault_init.go:77]: The godoc recommends not to use 0.
https://pkg.go.dev/golang.org/x/term#pkg-overview

```suggestion
	defer term.Restore(int(os.Stdin.Fd()), oldState)
```

File: pkg/ckecli/cmd/vault_init.go [hunk 2]
```diff
@@ -59,6 +60,30 @@ path "cke/*"
...
+
+	t := term.NewTerminal(screen, "")
+	p, err := t.ReadPassword(prompt)
```
Comment (morimoto-cybozu) [inline pkg/ckecli/cmd/vault_init.go:80]: A new terminal does not seem necessary.
I'm not sure whether `MakeRaw()` is necessary or not.

```suggestion
	fmt.Print(prompt)
	p, err := term.ReadPassword(int(os.Stdin.Fd()))
```

**Review by morimoto-cybozu — CHANGES_REQUESTED** [review]

File: pkg/ckecli/cmd/vault_init.go (hunk 2 above)
Comment (umezawatakeshi) [reply pkg/ckecli/cmd/vault_init.go:80]: according to my local poc just now, we have to print '\n' after ReadPassword. i.e. the following is enough:
```
	fmt.Print(prompt)
	p, err := term.ReadPassword(int(os.Stdin.Fd()))
	fmt.Println()
```

`MakeRaw` is required if we use `t.ReadPassword` but it does not seems required if we use `term.ReadPassword`.

**Review by morimoto-cybozu — APPROVED** [review]
LGTM

**Pull request merged**